lib/excision/excisions.ex

```diff
@@ -460,7 +462,7 @@ defmodule Excision.Excisions do
         %{active_classifier_id: classifier.id}
       )
 
-    {:ok, _} = update_classifier(classifier, %{promoted_at: DateTime.utc_now()})
+    update_classifier(classifier, %{promoted_at: DateTime.utc_now()})
```

Comment:
fix for type error: a consumer of promote_classifier wanted to match on :error, but promote_classifier would never return an error